src/table/body-cell/index.tsx

```diff
@@ -85,14 +79,8 @@ function TableCellEditable<ItemType>({
     <TableTdElement
       {...rest}
       nativeAttributes={tdNativeAttributes as TableTdElementProps['nativeAttributes']}
-      className={clsx(
-        className,
-        styles['body-cell-editable'],
-        resizableColumns && styles['resizable-columns'],
-        isEditing && styles['body-cell-edit-active'],
```

Comment:
This style is now added in the TableTdElement: `isEditing && !isEditingDisabled && styles['body-cell-edit-active']`